fly/commands/destroy_team.go

```diff
@@ -27,7 +28,7 @@ func (command *DestroyTeamCommand) Execute([]string) error {
 		return err
 	}
 
-	teamName := command.TeamName
+	teamName := string(command.TeamName)
```

Comment:
a minor nitpick - I think a method on `flaghelpers.TeamFlag` would be safer than casting here, in case the composition of that type changes from a simples tring later. Here's one example of this pattern: https://github.com/concourse/flag/blob/master/directory.go#L29

Reply:
Thanks for your review!
I fixed it 👍